blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -13,40 +14,34 @@ import { pipelineBranchesUnsupported } from './PipelinePage';
 
 const { object, array, func, string, any } = PropTypes;
 
-const EmptyState = ({ repoName }) => (
+const EmptyState = ({ repoName, t }) => (
     <main>
         <EmptyStateView iconName="branch">
-            <h1>Branch out</h1>
-
-            <p>
-                Create a branch in the repository <em>{repoName}</em> and
-                Jenkins will start testing your changes.
-            </p>
-
-            <p>
-                Give it a try and become a hero to your team.
-            </p>
-
-            <button>Enable</button>
+            <Markdown>
+                {t('EmptyState.branches', { 0: repoName })}
+            </Markdown>
```

Comment:
Lately some members on the team have expressed concerns about depending on too many React libs since it may limit our ability to upgrade React down the road. Do we really need markdown here or could we just have plug a few translation keys into a few HTML elements?